src/thorlabs_cube/driver/tcube/tpz.py

```diff
@@ -84,26 +80,29 @@ async def set_output_volts(self, voltage):
             :py:meth:`set_tpz_io_settings()<Tpz.set_tpz_io_settings>`
             method between the three values 75 V, 100 V and 150 V.
         """
+        if self.voltage_limit is None:
+            raise ValueError("Voltage limit is not set")
+
         if voltage < 0 or voltage > self.voltage_limit:
             raise ValueError(
                 "Voltage must be in range [0;{}]".format(self.voltage_limit)
             )
         volt = int(voltage * 32767 / self.voltage_limit)
-        payload = st.pack("<HH", 1, volt)
+        payload = st.pack("<HH", _CHANNEL, volt)
```

Comment:
```suggestion
        payload = st.pack("<HH", Tpz._CHANNEL, volt)
```
Use class constants.